api/src/wfl/build.clj

```diff
@@ -94,7 +95,7 @@
                    out-zip (str (util/unsuffix out-wdl ".wdl") ".zip")]
                (io/make-parents out-zip)
                (.renameTo in-wdl (io/file out-wdl))
-               (.renameTo in-zip (io/file out-zip)))
+               (if in-zip (.renameTo in-zip (io/file out-zip))))
              (finally (util/delete-tree directory)))))))
```

Comment:
Rather than generalizing the `cromwellify` workarounds …
Why not try to read from Warp at run time?
(Might need a new Cromwell submit path though.)

Reply:
I think reading from Warp is a great idea and would do a lot to help address this sorta icky stuff--I don't know if it is right to take on [GH-688](https://broadinstitute.atlassian.net/browse/GH-688) in this ticket since our deadline is so tight. Happy to give it a shot if you think that's the right call. Since Lantern is planning to add imports we could just remove the cromwellify workarounds too, whenever that happens.

Reply:
OK. Another time.